_data/engines.yml

```diff
@@ -1,4 +1,236 @@
+
+- name: Xl8
+  url: http://www.xl8.ai
+  languages: [ [en, ar], [en, cs], [en, da], [en, de], [en, el], [en, es], [en, es-419], [en, fi], [en, fr], [en, he], [en, hu], [en, id], [en, it], [en, ja], [en, ko], [en, ms], [en, my], [en, my-zawgyi], [en, nl], [en, no], [en, pl], [en, pt], [en, pt-BR], [en, ro], [en, ru], [en, sk], [en, sv], [en, th], [en, tr], [en, vi], [en, zh-cn], [en, zh-tw], [fi, da], [fi, en], [fi, no], [fi, sv], [fr, en], [no, da], [no, en], [no, fi], [no, sv], [sv, da], [sv, en], [sv, fi], [sv, no], [th, en], [zh-cn, en], [zh-tw, en], [es, en], [ko, en], [ko, ja], [ko, th], [ko, zh-cn], [ar, en], [nl, en], [pt, en], [es-419, en], [ja, en], [tr, en], [da, en], [da, fi], [da, no], [da, sv], [de, en], [pl, en] ]
+  info_url: https://api.xl8.ai/v1/trans/languages
+
+- name: Yandex
+  url: https://translate.yandex.com/
+  languages: [ af, am, ar, az, ba, be, bg, bn, bs, ca, ceb, cs, cy, da, de, el, en, eo, es, et, eu, fa, fi, fr, ga, gd, gl, gu, he, hi, hr, ht, hu, hy, id, is, it, ja, jv, ka, kk, km, kn, ko, ky, la, lb, lo, lt, lv, mg, mhr, mi, mk, ml, mn, mr, mrj, ms, mt, my, ne, nl, no, pa, pap, pl, pt, ro, ru, si, sk, sl, sq, sr, su, sv, sw, ta, te, tg, th, tl, tr, tt, udm, uk, ur, uz, vi, xh, yi, zh ]
+  info_url: https://yandex.com/dev/translate/doc/dg/concepts/api-overview.html
+
+- name: Yeekit (online platform)
+  url: http://www.yeekit.com
+  languages: [ ar, et, bg, pl, fa, da, de, ru, fr, fi, ko, nl, cs, hr, ro, ms, mn, mn-cyrl, bn, my, no, pt, ja, si, sv, sr, sk, sl, th, tr, ug, uk, es, he, el, hu, bo, it, hi, id, en, vi, yue, zh-cn ]
+  info_url: https://www.gtcom.com.cn/en/index.php?c=products&a=sandesc&id=61
+
+- name: Yeekit (MT API)
+  url: http://api.yeekit.com
+  languages: [ en, zh ]
+  info_url: http://api.yeekit.com/mannual.php#docs
+
+- name: Youdao Translate
+  url: http://fanyi.youdao.com
+  languages: [ af, am, ar, az, be, bg, bn, bs, ca, ceb, co, cs, cy, da, de, el, en, eo, es, et, eu, fa, fi, fj, fr, fy, ga, gd, gl, gu, ha, haw, he, hi, hr, ht, hu, hy, id, ig, is, it, ja, jw, ka, kk, km, kn, ko, ku, ky, la, lb, lo, lt, lv, mg, mi, mk, ml, mn, mr, ms, mt, mww, my, ne, nl, no, ny, otq, pa, pl, ps, pt, ro, ru, sd, si, sk, sl, sm, sn, so, sq, sr-Cyrl, sr-Latn, st, su, sv, sw, ta, te, tg, th, tl, tlh, to, tr, ty, uk, ur, uz, vi, xh, yi, yo, yua, yue, zh-cn, zh-tw, zu ]
+  info_url: https://ai.youdao.com/DOCSIRMA/html/%E8%87%AA%E7%84%B6%E8%AF%AD%E8%A8%80%E7%BF%BB%E8%AF%91/API%E6%96%87%E6%A1%A3/%E6%96%87%E6%9C%AC%E7%BF%BB%E8%AF%91%E6%9C%8D%E5%8A%A1/%E6%96%87%E6%9C%AC%E7%BF%BB%E8%AF%91%E6%9C%8D%E5%8A%A1-API%E6%96%87%E6%A1%A3.html
+
+- name: Wordlingo
+  url: https://www.worldlingo.com/en/products/worldlingo_api.html
+  languages: [ af, ar, az, be, bg, bn, ca, cs, cy, da, de, el, en, eo, es, et, eu, fa, fi, fr, ga, gl, gu, ha, he, hi, hr, ht, hu, id, is, it, ja, ka, kn, ko, la, lt, lv, mk, ms, mt, nl, no, pl, pt, ro, ru, sk, sl, so, sq, sr, sv, sw, ta, te, th, tl, tr, uk, ur, vi, yi, zh_cn, zh_tw ]
```

Comment:
Should be `-` not `_`, but I guess we could make the other script deal with that.

Reply:
Fixed